### PR TITLE
ParseRSSFeed Task

### LINKED ISSUE #853: Create RSS Tasks for Task Library

### PR DESCRIPTION
**Pull request by joshmeek**
**Thanks for contributing to Prefect!**

Please describe your work and make sure your PR:

- [x] adds new tests (if appropriate)
- [x] updates `CHANGELOG.md` (if appropriate)
- [x] updates docstrings for any new functions or function arguments, including `docs/outline.toml` for API reference docs (if appropriate)

## What does this PR change?
Adds a `ParseRSSFeed` task for parsing an RSS feed from a remote URL
Closes #853 

## Why is this PR important?
It is a common task and one I wanted personally :)

### CODE REVIEW

File: src/prefect/tasks/rss/feed.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            - FeedParserDict: A dictionary representing the information from the
+                parsed feed. The object is accessable through indexing and attributes.
+
+        Raises:
+            - ValueError: if `feed_url` is `None`
+        """
+        if not feed_url:
+            raise ValueError("The feed_url must be provided.")
+
+        return feedparser.parse(feed_url)
```
Comment (cicdw) [inline src/prefect/tasks/rss/feed.py]: what is the return value of this?

Comment (cicdw) [reply src/prefect/tasks/rss/feed.py]: I guess my real question is: do you know if the output here is cloud pickle-able?

Comment (joshmeek) [reply src/prefect/tasks/rss/feed.py]: It's `feedparser.FeedParserDict` but let me check if it is cloud pickle-able

Comment (joshmeek) [reply src/prefect/tasks/rss/feed.py]: Worked like a charm

Comment (cicdw) [reply src/prefect/tasks/rss/feed.py]: Awesome! 🚀

**Comment by cicdw** [comment]
@joshmeek you need to add a `pytest.importorskip` statement to the `__init__.py` of your rss/ tests subdirectory.  I love that `test-vanilla-prefect` job haha

File: tests/tasks/rss/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import pytest
+
+pytestmark = pytest.importorskip("feedparser")
```
Comment (cicdw) [inline tests/tasks/rss/__init__.py]:
```suggestion
pytest.importorskip("feedparser")
```
to prevent more work in #814

**Pull request merged**